ydb/library/workload/kv_workload.h

```diff
@@ -103,10 +107,13 @@ class TKvWorkloadGenerator final: public TWorkloadQueryGeneratorBase<TKvWorkload
 
     TQueryInfo FillKvData() const;
     TVector<TRow> GenerateRandomRows(bool randomValues = false);
+    TVector<TRow> GenerateSequentialRows();
```

Comment:
I propose to reuse old workloads and add `--sequential` flag

For here we may also just transform existing GenerateRandomRows -> GenerateRows(bool randomValues = false, bool sequential = false, ...)